Formula/mongodb-community.rb

```diff
@@ -31,6 +31,7 @@ def mongodb_conf; <<~EOS
       dbPath: #{var}/mongodb
     net:
       bindIp: 127.0.0.1
+      ipv6: true
```

Comment:
@tstirrat15  - Thanks for the PR! It makes sense to me that we would want to listen on the IPv6 interfaces too.

I checked with some of the folks who work closely with networking in the product, and they said that for this to work right, we probably should either 1) add `::1` to the `bindIp` list, or 2) eliminate `bindIp` entirely, since MongoDB only listens to localhost by default in modern versions. Personally, I'm leaning towards the former, under the view that explicit is better than implicit.

Reply:
Yeah - when I was looking at `bindIp` I think it might need to include `::1` as well. When I was troubleshooting this on my partners' computer the changes I made to this PR didn't seem to be sufficient.

Removing `bindIp` did fix things. My only concern would have been that it would default to `0.0.0.0` which could be problematic from a security perspective, but if it binds to `localhost` only I might lean in the direction of keeping the config minimal and removing `bindIp`.

Reply:
@tstirrat15  - 

If you don't mind, I think please add the ::1 for now and keep it explicit. My reasoning is that it may not yet be common knowledge that modern MongoDB listens only on localhost by default, and I don't want developers looking at this file to draw the incorrect conclusion that their homebrew deployment is insecure when they don't see a `bindIp` value, or be unsure about it and need to dig it out of the docs where the right answer is version dependent. I think there is something reassuring seeing those explicit localhost entries. The improved default is great and the right thing to do, but should perhaps be viewed as more of a backstop to prevent mistakes, rather than as something to be relied upon for proper configuration.

Reply:
Are there automated tests for this or should I go figure out how to validate it?

Reply:
There are not currently automated tests. If you can validate, that would be helpful.

Reply:
I made those changes on my partner's computer and ran a client against it and it seemed to work.

Reply:
I also tested locally and it worked for me as well.

Reply:
I suggest removing the bindIp line. For one thing, binding to 127.0.0.1 and ::1 isn't exactly the default behavior of binding to localhost. 

Rather I suggest adding comments like this:

```
# by default, we bind to (listen on) localhost only. to bind to all network interfaces uncomment the following line. See https://docs.mongodb.com/manual/core/security-mongodb-configuration/#ip-binding for details.
#       bindIpAll: true
```

Reply:
FTR; Not having the `bindIp` line there at all [will](https://github.com/mongodb/mongo/blob/ae8042e3e66b160b2a47c4b7f071c804e9271768/src/mongo/transport/transport_layer_asio.cpp#L1042-L1046) result in `127.0.0.1,::1`, but IMO it doesn't hurt to be explicit here.

Reply:
Let's go with explicit for now, as the PR currently has it.